guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -27,52 +27,64 @@ endif::[]
+. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
 ifndef::satellite,orcharhino[]
 Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
 endif::[]
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register your host through.
+. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
 +
 [NOTE]
 ====
 A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selected in the {ProjectWebUI} as the host's content source.
 ====
 ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to your host.
+. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
 endif::[]
-. If you want to use `wget` to register your host to {Project}, select the *Use wget* option.
-By default, {Project} generates a `curl` command.
-. If your host does not trust the SSL certificate of your {ProjectServer}, select the *Insecure* option.
-During the first call, your host downloads the CA file from {Project}.
-Your host will use this CA file to connect to {ProjectServer} with all future calls.
+. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+During this first call, hosts download the CA file from {Project}.
+Hosts will use this CA file to connect to {Project} with all future calls making them secure.
 +
 {Team} recommends that you avoid insecure calls.
 +
-If an attacker, located in the network between {Project} and your host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from your host and the JSON Web Tokens (JWT).
-Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access your host using the SSH key.
+If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
+Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
 +
-Instead, you can manually copy and install the CA file on each host before registration.
+Instead, you can manually copy and install the CA file on each host before registering the host.
 +
 To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
 +
-Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on your host and enter the following commands:
+Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
```

Comment:
You use the singular "host" above and plural "hosts" here. Perhaps change "hosts" to "each host". That makes the change in number less noticeable and still conveys that the CA certificate must be copied to each host.